internal/hooks/module_source_local.go

```diff
@@ -0,0 +1,18 @@
+package hooks
+
+import (
+	"context"
+
+	"github.com/hashicorp/hcl-lang/decoder"
+	"github.com/zclconf/go-cty/cty"
+)
+
+func (h *Hooks) LocalModuleSources(ctx context.Context, value cty.Value) ([]decoder.Candidate, error) {
```

Comment:
I'm assuming this would be eventually part of a separate PR? 👀

Reply:
I added this as a placeholder to showcase the registration of hooks in `AppendCompletionHooks`. Without any hooks we wouldn't be able to create `hooks.Hooks{}`.

But I can remove it if you want? And defer everything to the next PR